codalab/worker/worker_run_state.py

```diff
@@ -628,13 +727,19 @@ def finalize_run(self, run_state):
         """
         Prepare the finalize message to be sent with the next checkin
         """
+        log_bundle_transition(
+            bundle_uuid=run_state.bundle.uuid,
+            previous_state=run_state.stage,
+            next_state=RunStage.FINALIZING,
+        )
         if run_state.is_killed:
             # Append kill_message, which contains more useful info on why a run was killed, to the failure message.
             failure_message = (
                 "{}. {}".format(run_state.failure_message, run_state.kill_message)
                 if run_state.failure_message
                 else run_state.kill_message
             )
+            logger.error(f'{run_state.bundle.uuid} is killed: {failure_message}')
```

Comment:
Killing a bundle is not necessarily an error.

Reply:
change to info